extensions/blocks/podcast-player/style.scss

```diff
@@ -164,12 +159,15 @@ $block-border-color: $dark-gray-100;
 		color: inherit;
 		transition: none;
 
-		.jetpack-podcast-player__episode.is-active & {
-			.is-error & {
-				padding-bottom: 0; // Make space for the error element that will be appended.
-			}
+		&:hover,
+		&:focus {
+			color: $text-color-hover;
 		}
 	}
+
+	.is-error &.is-active .jetpack-podcast-player__episode-link {
+		padding-bottom: 0; // Make space for the error element that will be appended.
+	}
```

Comment:
My dev environment is acting up so I can't test to confirm at the moment, but is this part now not scoped correctly? Should this be _outside_ of this `.jetpack-podcast-player__episode {` scope?

Reply:
This one is correct in this form. The `&` is a reference to the scope, so in this case the `.jetpack-podcast-player__episode`.

The output of this is:

```css
.is-error .jetpack-podcast-player__episode.is-active .jetpack-podcast-player__episode-link {
    padding-bottom: 0;
}
```